app/assets/javascripts/tagging.js

```diff
@@ -140,7 +140,7 @@ function promptTag(val) {
       break;
 
     default:
-      addTag(expr);
+      addTag(val);
```

Comment:
This change is not directly related to the 'locked' tag but is required in the `promptTag` function.

Reply:
Ah, was it a bug from before? Cool. Thanks!